apps/test/unit/lib/kits/maker/FakeChromeSerialPort.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @file Stand-in for chrome-serialport library, for use in tests.
+ */
+class SerialPort {}
+
+let deviceList = [];
+
+export default {
+  SerialPort,
+
+  isInstalled(callback) {
+    callback(null);
+  },
+
+  // Give back a valid list of serial port info
+  list(callback) {
+    callback(null, deviceList);
+  },
+
+  test_setDeviceList(list) {
+    deviceList = list;
+  },
+
+  test_reset() {
+    deviceList = [];
+  }
+};
```

Comment:
Is this a "fake" or a "mock"? I can never remember the difference. In any case, we should perhaps have a naming convention for these special functions for controlling the fake/mock (`test_setDeviceList` and `test_reset` in your code). I've been using `mockDoSomething()` in my tests. I've seen other test frameworks expose a "mock" object with these controls, like `SerialPort.mock.setDeviceList()`.